ujin/programmers/day03/자릿수-더하기.js

```diff
@@ -0,0 +1,8 @@
+function solution(n) {
+  let answer = 0;
+  let nStr = String(n);
```

Comment:
값이 변하지 않는 경우 `const` 로 선언하는 것이 예상치 못한 버그나 오류를 방지할 수 있습니다.
```suggestion
  const nStr = String(n);
```